openmc/mgxs/library.py

```diff
@@ -183,6 +184,8 @@ def domains(self):
                 return self.openmc_geometry.get_all_material_cells()
             elif self.domain_type == 'universe':
                 return self.openmc_geometry.get_all_universes()
+            elif self.domain_type == 'mesh':
+                raise ValueError('Unable to get domains for Mesh domain type')
```

Comment:
It could, but I don't see a way to get the mesh since the geometry doesn't have it. Do you think we should add the mesh as an attribute to the library?